### PR TITLE
Cost component

### PR DESCRIPTION
**Pull request by Pfdiamond13**
@krise3 please review, this PR includes 

- new Cost component
- new test suite for Cost component
- small refactors on Checkout/Checkout.test

### CODE REVIEW

File: client/src/components/Checkout.jsx [hunk 1]
```diff
@@ -46,11 +47,11 @@ class Checkout extends React.Component {
...
-        <div>{this.state.rooms[0].avg_review}avg reviews</div>
-        <div>{this.state.rooms[0].reviews}reviews</div>
+        <div><Cost room={this.state.room[0]}/></div>
+        <div>{this.state.room[0].avg_review}avg reviews</div>
+        <div>{this.state.room[0].reviews}reviews</div>
         <div>Dates</div>
-        <div>{this.state.rooms[0].guests} guests</div>
+        <div>{this.state.room[0].guests} guests</div>
         <div>Booked Button</div>
         <div>You are the best</div>
```
Comment (krise3) [inline client/src/components/Checkout.jsx]: Is this leftover testing text?

Comment (Pfdiamond13) [reply client/src/components/Checkout.jsx]: Good catch, removed!

File: client/src/components/Checkout.jsx [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import axios from 'axios';
 import styled from 'styled-components';
+import Cost from './Cost';
 
 const Wrapper = styled.section`
```
Comment (krise3) [inline client/src/components/Checkout.jsx:6]: Nice styled component!

File: client/src/components/Cost.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  overflow-wrap: break-word;
+  line-height: 1.44444em; 
+
+`;
+
+const NightWrapper = styled.section`
+  float: left;
+  font-size: 12px !important;
+  font-weight: 600 !important;
+
```
Comment (krise3) [inline client/src/components/Cost.jsx]: Empty line breaks before the closing back tick for both NightWrapper and CostWrapper

File: client/spec/Checkout.test.js [hunk 4]
```diff
@@ -1,17 +1,29 @@
...
+
+  xit('Should have one <Cost /> component', () => {
+    const wrapper = mount(<Checkout />);
+    expect(wrapper.find(Cost)).toHaveLength(1);
   });
+
+  xit('Should fetch rooms', () => {
+    const room = [{roomId: '1'}];
+    const resp = {data: room};
+    axios.get.mockImplementation(() => Promise.resolve(resp));
```
Comment (krise3) [inline client/spec/Checkout.test.js:26]: Looking good so far! Concise tests, nice use of 'x' flags

File: client/spec/Cost.test.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Cost from '../src/components/Cost';
+
+const setup = () => {
+  const wrapper = shallow(<Cost room={ {cost:'216'} }/>);
+  return wrapper;
```
Comment (krise3) [inline client/spec/Cost.test.js:7]: Thanks for not returning an expression here. Also, hooray for code reusability!

**Review by krise3 — APPROVED** [review]
@Pfdiamond13 All extraneous code and style deviations confirmed squashed, and everything else checks out. Nice work!

**Pull request merged**